### ISSUE TITLE
Test: Mentors Page - Search & Filters

### ISSUE THREAD

**Issue by MitaliShah** [WomenCodingCommunity/WomenCodingCommunity.github.io#219]
### Description

1. e2e test to test the Mentors  Search & Filters feature 
2. https://womencodingcommunity.com/mentors
3. Please check [the previous](https://github.com/dricazenck/london/blob/main/e2e-tests/cypress/e2e/mentors_page/mentors_page_search.feature) test scenario for reference:

**Comment by khairahscorner** [comment]
Unassigning this issue, as it has been pending without a merge for some time

**Comment by gillian-black** [comment]
@khairahscorner do any of these older tickets need to be done? I'd like to get more testing experience so happy to jump on any of them.

**Comment by khairahscorner** [comment]
@gillian-black absolutely! Please feel free to pick any of the issues and leave a comment that you'd like to work on them so that they can be assigned to you🚀

**Comment by khairahscorner** [comment]
Unassigning as this has been pending for a while

**Comment by dricazenck** [comment]
For now let's close, because we are working in the platform project and this will covered there.
